styles.css

```diff
@@ -0,0 +1,158 @@
+html {
+	font-family: "Trebuchet MS", sans-sarif;
+	box-sizing: border-box;
+	font-size: 10px;
+}
+
+body {
+	background-color:#0a415c;
+	margin: 0;
+	padding: 0;
+	color:#cfd2d4;
+}
+
+h2 {
+	font-size: 3rem;
+}
+
+h3 {
+	font-size: 2rem;
+}
+
+p {
+	font-size: 1.5rem;
+}
+
+nav {
+	display: flex;
+	flex-direction: row;
+	flex-wrap: nowrap;
+	justify-content: space-between;	
+	align-items: center;
+}
+
+.nav-bar {
+	width: 20%;
```

Comment:
this width is not needed - the width of the element is based on the total width of the elements within it